testing/integration_test.go

```diff
@@ -72,16 +73,18 @@ set -eo pipefail
 set -x
 # Install git
 retry apt-get update >/dev/null
-retry apt-get -y -q install git build-essential >/dev/null
+retry apt-get -y -q install git {{if eq .BinaryHost ""}}build-essential{{end}} >/dev/null
```

Comment:
I think "{{if eq .BinaryHost ""}}" can be just "{{if .BinaryHost}}" like we do for "{{if .NVMMirror}}" below?